ros_gz_bridge/src/convert/marine_acoustic_msgs.cpp

```diff
@@ -0,0 +1,157 @@
+// Copyright 2025 Honu Robotics
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <cmath>
+#include "gz/math/Vector3.hh"
+#include "gz/msgs/convert/Vector3.hh"
+#include "convert/utils.hpp"
+#include "ros_gz_bridge/convert/marine_acoustic_msgs.hpp"
+
+namespace ros_gz_bridge
+{
+
+template<>
+void
+convert_ros_to_gz(
+  const marine_acoustic_msgs::msg::Dvl & ros_msg,
+  gz::msgs::DVLVelocityTracking & gz_msg)
```

Comment:
include msgs headers

Reply:
Added in [89b14aa](https://github.com/gazebosim/ros_gz/pull/789/commits/89b14aa10731aed70f7ae7acdbe1c5ce8698f6f4).